chain/validator-config.toml

```diff
@@ -6,36 +6,26 @@ keys_path = "/home/parity/.local/share/io.parity.ethereum/custom/keys"
 auto_update = "none"
 # Disables auto downloading of new releases. Not recommended.
 no_download = true
-# Save pending local transactions to disk to be restored whenever the node restarts.
-no_persistent_txqueue = false
 
 [network]
 port = 30300
-# Parity will maintain at most 100 peers.
-max_peers = 100
 
 [rpc]
 interface = "all"
 
 [account]
 # File should contain passwords to unlock your accounts. One password per line.
-# password = ["/home/parity/.local/share/io.parity.ethereum/custom/pass.pwd"]
-# List of addresses to unlock for the duration of execution.
-# unlock = ["0xAddress"]
+password = ["/home/parity/.local/share/io.parity.ethereum/custom/pass.pwd"]
```

Comment:
Where is unlock?

Reply:
You don't need this for a validator.
`git cat-file -p f7549b50f7aad00c2636925c25ef450e384e1e57:chain/parity_wrapper.sh` shows that this options was only added for the role of a participant.